src/CheckoutSdk/Marketplace/MarketplaceFileType.cs

```diff
@@ -0,0 +1,33 @@
+﻿namespace Checkout.Marketplace
+{
+    public class MarketplaceFileType
+    {
+        public string Value { get; private set; }
+
+        private MarketplaceFileType(string type)
+        {
+            Value = type;
+        }
+
+        public static MarketplaceFileType Passport
```

Comment:
What do you think about refactoring this and the remaining methods to `expression-bodied` properties? Like this:

`public static MarketplaceFileType Passport => new MarketplaceFileType("passport")`

Reply:
Nice!